src/pages/services.tsx

```diff
@@ -46,61 +46,61 @@ const SERVICE_BLOCKS: Omit<ServiceProps, 'align'>[] = [
   {
     title: 'Websites',
     content:
-      "If you're a small vegan activist, organization, or business with little to no funding, you're the ones we want to help succeed the most! In many cases we support folks that either can't pay for website design, development or have little knowledge or time to build it themselves.",
+      "If you're an independent activist or organization with little to no funding, you're the ones we want to help. We support advocates that have few resources dedicated to website design, development, and/or have limited knowledge or time to build it themselves.",
     icon: webIcon,
     iconBgColor: 'magenta',
     iconAccentColor: 'red',
     button: {
-      text: 'Apply for a free website!',
+      text: 'Apply for a free website',
       href: 'https://veganhacktivists.org/playground',
     },
   },
   {
     title: 'Design',
     content:
-      'We have a dedicated design team that can help you with your branding, logo, social media banners, icons, funding documents, website design, and more! Note that requests are free but accepted on a case-by-case basis, we encourage you to submit your request to us below!',
+      "We have a dedicated team to help design your brand's identity, logo, social media assets, website design, and more. We collaborate with you to understand your core values, purpose, and goals, so that we can best curate your digital presence.",
```

Comment:
```suggestion
      "We have a dedicated team to help design your brand's identity, logo, social media assets, website, and more. We collaborate closely with you to understand who you are and what your needs are, so that we can best curate your digital presence.",
```